src/applications/disability-benefits/all-claims/content/ptsdClassification.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+export const getPtsdClassification = (formData, formType) => {
+  const classifications = formData['view:selectablePtsdTypes'];
```

Comment:
Should we use the `_.get` util in `../../../platform/utilities/data` to safely traverse this object?

Reply:
Yes, I agree.